Documentation/ApiOverview/Events/EventDispatcher/Index.rst

```diff
@@ -286,10 +298,16 @@ The tag name :yaml:`event.listener` identifies that a listener should be registe
 
 The custom PHP class :php:`\MyVendor\MyExtension\EventListener\NullMailer`
 serves as the listener whose :php:`handleEvent()` method is called, once the
-:yaml:`event` is dispatched. The :yaml:`identifier` is a common name, so
+:yaml:`event` is dispatched. The name of the listened event is specified as
```

Comment:
To be precise: A method of a listener can have multiple type declarations for an event, like

    __invoke(Event1|Event2 $event)

Reply:
That would maybe be another addition, you could create it as its own PR to not further make this more complicated? IMO addressing implementing multiple events needs a few sentences wihch are beyond my scope at this point (adressing to override event listeners).